history-of-the-quakers-v2/original/08-book-3-chapter-14.adoc

```diff
@@ -250,14 +261,18 @@ As soon as this order was obtained, it was quickly brought to Scarborough,
 and delivered to the governor, who upon receipt thereof immediately discharged him,
 and gave him the following passport.
 
-"`Permit the bearer hereof George Fox, late a prisoner here,
+[.embedded-content-document.legal]
+--
+
+Permit the bearer hereof George Fox, late a prisoner here,
 and discharged by his majesty`'s order, quietly to pass about his lawful occasions,
 without molestation.
 Given under my hand at Scarborough castle this 1st day of September, 1666.
 
-"`JORDAN CROSSLANDS,
+[.signed-section-signature]
+Jordan Crosslands, Governor of Scarborough Castle.
```

Comment:
Hmmm... I waffled a little bit on this, but I think it feels slightly better to me to move this "Governer of Scarborough Castle." line down on it's own, and make it the [sscc].